saraho1123/currency-exchange
# Feature/display currency cards

## Is this a fix or a feature?
feature

## What is the change?
+ displays cards with conversion information
+ had to update Form and App so that the api call could use the user input to make the call. 
+ then I had to manipulate the data from the api call to get the conversion amount

## What does it fix?
Note that there is a troubleshooting branch that I am not merging. I did not want to lose work that was working as I was troubleshooting how to update state at the correct times. It turned out that I needed a conditional rendering! Whew!

## Where should the reviewer start?
line 11 of App.js
line 4 of CurrencyCard.js
line 10 of ExchangeContainer.js
line 10 of Form.js

## How should this be tested?
+ clone down
+ cd into project
+ run `npm install` in terminal
+ run `npm start` in terminal
Form should appear. It can be filled out and button clicked, and user taken to `/currency-cards`.
The data from the user inputs, api data and manipulated api data would render
This can be done multiple times by click 'Get A New Currency Exchange' at the top of the page
NOTE: styling and organization of data is incomplete at this point.

## Relevant Tickets
+ #3 
+ #4 
+ #5 
+ #6
